088-rebalance-progress-status.md

```diff
@@ -0,0 +1,374 @@
+# Partition Rebalance Progress Status
```

Comment:
Should be more "Cluster rebalance progress status" (or rebalancing) ... not sure if "partition" (even using the singular) sounds really fine because it's about a cluster rebalancing and about moving one or more partitions across the cluster.